.github/workflows/container-tests.yml

```diff
@@ -1,20 +1,17 @@
+name: Container tests
+
 on:
-  issue_comment:
-    types:
-      - created
+  pull_request:
+    branches:
+      - 'master'
 jobs:
   distgen-check:
     name: "Check distgen generated files"
     runs-on: ubuntu-20.04
-    if: |
-      github.event.issue.pull_request
-      && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
-      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
         with:
-          ref: "refs/pull/${{ github.event.issue.number }}/head"
```

Comment:
I think that we should checkout the repository on the last commit to the head branch of the pull request, which should be stored in this variable ${{ github.event.pull_request.head.sha }} [1].

[1] https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Right, sorry.